.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "GameServerApp/Content/Data/LeagueSandbox-Default"]
 	path = GameServerApp/Content/Data/LeagueSandbox-Default
 	url = https://github.com/LeagueSandbox/LeagueSandbox-Default.git
-	branch = indev
+	branch = spell-refactor-phase-2
```

Comment:
This should probably stay as indev, and instead wait for the PR in the other repo to be accepted.